### ISSUE TITLE
RSS feeds not generating properly

### ISSUE THREAD

**Issue by skoobasteeve** [jekyll-octopod/jekyll-octopod#70]
Hello,
I'm new to the Octopod project, so let me first say how much I appreciate all the work you've put in. Currently building out a site for my new podcast and am excited to do so with a static site :). 

The issue I'm currently having is with RSS feeds. They seem to only contain the HTML of the post page and not any actual RSS data. I compared with the demo site to be sure. You can see my work-in-progress deployment here: https://pod.raylyon.info

Steps to Reproduce:
1. `octopod episode --title "1991: The Fall of The Soviet Union"`
2. Fill in remaining metadata as seen in attached file
3. `octopod build`
4. `octopod deploy`

Can you help me better understand how Octopod is generating the RSS feeds? Am I missing something in the _config.yml file or the post metadata? Let me know if you have any ideas. I attached my _config file, the problem RSS feed as well as an example post from my site. (changed extension to txt so github would let me upload).

Thank you!

[episodes.mp3.rss.txt](https://github.com/jekyll-octopod/jekyll-octopod/files/4524920/episodes.mp3.rss.txt)

[2020-04-23-1991_-the-fall-of-the-soviet-union_.md.txt](https://github.com/jekyll-octopod/jekyll-octopod/files/4524923/2020-04-23-1991_-the-fall-of-the-soviet-union_.md.txt)

[_config.yml.txt](https://github.com/jekyll-octopod/jekyll-octopod/files/4524936/_config.yml.txt)

**Comment by chetwisniewski** [comment]
I am having the same issue, live at https://www.linuxatwork.org

**Comment by haslinger** [comment]
Thanks for bringing this up, as the behavior is not showing up on my own podcasts. 
I will look into this a.s.a.p.
Update: I am really surprised by this, as there is no magic involved and the jsonfeeds are still working, we are just using layout/feed.xml as a layout...

**Comment by chetwisniewski** [comment]
Could it be Jekyll 4.0?

Chester Wisniewski
chetw@zuzax.com
________________________________

**Comment by haslinger** [comment]
@chetwisniewski @0scubasteve0 Could you please try the following quick fix: 
In your local project root directoy remove the line `{{ content }}` from the `episodes.xxx.rss` files. 
That should resolve the issue. Does it?

**Comment by skoobasteeve** [comment]
@haslinger That fixed it for me! Removed the line, ran `octopod build` and the RSS feeds are working. Thank you for the quick response!

Any idea why this would happen with us and not others?

ruby 2.5.5
gem 2.7..6.2
bundler 2.1.4
jekyll 4.0.0

**Comment by haslinger** [comment]
@0scubasteve0 nice, a new version will be released soon, but no need to do any upgrades for you as this is the only change. This must have been a bug forever (probably before I took over the library), but one that didn't show any outcome until Jekyll 4, at least that broke it for me as well. You two are probably the only new users since the upgrade last month.

**Comment by skoobasteeve** [comment]
Great to hear, thanks again for your work. I have a couple questions about post formatting but I'll create another issue.

**Comment by chetwisniewski** [comment]
Mine is now generating duplicates with a 1?

```-rw-r--r--  1 chetwisniewski chetwisniewski 13437 Apr 24 17:41 episodes1.m4a.rss
-rw-r--r--  1 chetwisniewski chetwisniewski 13421 Apr 24 17:41 episodes1.mp3.rss
-rw-r--r--  1 chetwisniewski chetwisniewski 13464 Apr 24 17:41 episodes1.ogg.rss
-rw-r--r--  1 chetwisniewski chetwisniewski 13437 Apr 24 17:41 episodes.m4a.rss
-rw-r--r--  1 chetwisniewski chetwisniewski 13421 Apr 24 17:41 episodes.mp3.rss
-rw-r--r--  1 chetwisniewski chetwisniewski 13464 Apr 24 17:41 episodes.ogg.rss
```
What would cause that? How do I fix it? I don't have a template for the 1's?
Is this for pagination?

**Comment by haslinger** [comment]
@chetwisniewski  Please do use new issue for unrelated topics to make it easier to understand for people who chime in later.
That is pagination and no need to fix. Have a look at the paged_feed_page_generator to understand the mechanism: https://github.com/jekyll-octopod/jekyll-octopod/blob/master/lib/jekyll/paged_feed_page_generator.rb and configure the episodes_per_feed_page setting.

**Comment by haslinger** [comment]
The original issue is fixed with ea25c08d34 and from jekyll-octopod-0.9.23 on.
